content/en/data_streams/_index.md

```diff
@@ -13,170 +17,21 @@ Data Streams Monitoring provides a standardized method for teams to understand a
 * Pinpoint faulty producers, consumers or queues, then pivot to related logs or clusters to troubleshoot faster.
 * Prevent cascading delays by equipping service owners to stop backed up events from overwhelming downstream services.
 
-## Setup
+### Setup
 
-{{< programming-lang-wrapper langs="java,go,dotnet" >}}
```

Comment:
Suggestion, maybe we move this to the end of the overview section? It feels weird to have the table right at the start of setup, it would probably be nicer to have the runtime buttons directly.